cSpell.json

```diff
@@ -0,0 +1,159 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Avalonia",
+		"binlog",
+		"Blazor",
+		"blockquotes",
+		"Bootstrapper",
+		"brotli",
+		"browsersync",
+		"Callout",
+		"chatops",
+		"codebases",
+		"Codespace",
+		"Codespaces",
+		"Contoso",
+		"Cupertino",
+		"customizability",
+		"databinding",
+		"datacontext",
+		"datagrid",
+		"devs",
+		"Dismissable",
+		"Docfx",
+		"ellipsize",
+		"Expando",
+		"Framebuffer",
+		"Geolocation",
+		"globbing",
+		"Gyrometer",
+		"Hanja",
+		"hectopascals",
+		"laggy",
+		"layouter",
+		"layouting",
+		"Listview",
+		"LLRM",
+		"mergeable",
+		"MSAL",
+		"msbuild",
+		"MVVM",
+		"NETSDK",
+		"netstandard",
+		"oidc",
+		"Omnisharp",
+		"overscroll",
+		"Packt",
+		"parameterless",
+		"pickable",
+		"Pluralsight",
+		"POSIX",
+		"reentrantly",
+		"roadmap",
+		"sandboxed",
+		"Segoe",
+		"Silverlight",
+		"Skia",
+		"skiasharp",
+		"Storyboarded",
+		"struct",
+		"Syncfusion",
+		"templatable",
+		"templating",
+		"timespan",
+		"Tizen",
+		"TLDR",
+		"triaging",
+		"UI's",
+		"unmanaged",
+		"Uno's",
+		"unoplatform",
+		"UWP's",
+		"VSIX",
+		"WEBASSEMBLY",
+		"walkthrough"
+	],
+	"ignoreWords": [
```

Comment:
ℹ️ Quick note: `words` can become spellCheck suggestions, whereas `ignoreWords` will not.